doc/classes/EditorImportPlugin.xml

```diff
@@ -241,7 +241,78 @@
+				This function can only be called during the [method _import] callback and it allows manually importing resources from it. This is useful when the imported file generates external resources that require importing (as example, images). Custom parameters for the ".import" file can be passed via the [param custom_options]. Additionally, in cases where multiple importers can handle a file, the [param custom_importer] can be specified to force a specific one. This function performs a resource import and returns immediately with a success or error code. [param generator_parameters] defines optional extra metadata which will be stored as [code skip-lint]generator_parameters[/code] in the [code]remap[/code] section of the [code].import[/code] file, for example to store a md5 hash of the source data. For example:
+				[codeblocks]
+				[gdscript]
+				func _import(source_file, save_path, options, r_platform_variants, r_gen_files):
+				    # Make textures folder if required.
+				    var source_file_folder = source_file.rsplit("/", true, 1)[0]
+				    var textures_folder: String = "%s/textures" % source_file_folder
+				    DirAccess.make_dir_recursive_absolute(textures_folder)
+
+				    # Simulate image export by copying the source file to the textures folder.
+				    var export_path: String = "%s/icon.png" % textures_folder
+				    var export_image_file: FileAccess = FileAccess.open(export_path, FileAccess.WRITE)
+				    export_image_file.store_buffer(FileAccess.get_file_as_bytes("res://game/my_icon.png"))
+				    export_image_file.close()
+
+				    # Ensure file visibility for ResourceLoader later.
+				    var editor_file_system: EditorFileSystem = EditorInterface.get_resource_filesystem()
+				    editor_file_system.update_file(textures_folder)
+				    editor_file_system.update_file(export_path)
+
+				    # Append exported file as external resource.
+				    var err: Error = self.append_import_external_resource(export_path)
+				    if err != OK:
+				        return err
+
+				    # Load exported image as a Texture2D.
+				    var export_image_tex: Texture2D = ResourceLoader.load(export_path, "Texture2D", ResourceLoader.CACHE_MODE_REPLACE)
+				    if export_image_tex == null:
+				        return ERR_SCRIPT_FAILED
+
+				    ...
```

Comment:
I think there is some value in indicating where more processing should happen, but agree that it shouldn't break copy-pasting snippets. [doc/classes/RichTextLabel.xml](/godotengine/godot/tree/master/doc/classes/RichTextLabel.xml#L237) uses `# ...` in this situation, which I think would be a good compromise?

I'll tangentially note that [doc/classes/Tween.xml](/godotengine/godot/tree/master/doc/classes/Tween.xml#L113) uses ellipses extensively in code sections, including in places like function calls where it would be helpful to have complete examples. That might be something to improve in a separate PR, though. (I also desperately want to reword "run parallelly" there to "run in parallel". I wish I could dedicate a week or two to just clean up all of the documentation, or at least the really important pages like Tween which people will most often reference.)

Reply:
A comment with the ellipsis is much better, yeah.